### DISCUSSION TITLE
Regular HLS call?

### DISCUSSION THREAD

**Discussion by michaelpj** [haskell/haskell-language-server#2914] [General]
I think it would be nice to have an occasional but regular call where we can have a chat about what we're up to. This would be especially nice since we're about to get some GSOC students, and they would probably benefit from that (I'll likely talk to them anyway, but having more people is nice!).

What do people think? Maybe every 2-3 weeks?

Tagging a random selection of people who I think might be interested: @pepeiborra @wz1000 @fendor @Ailrun @July541 @hasufell @alanz

**Comment by hasufell** [comment]
Sounds good. I'd probably put it at every month and otherwise as-needed (e.g. big PRs or imminent releases).

**Comment by Ailrun** [comment]
Sounds good to me as well.

**Comment by michaelpj** [comment]
I made a when2meet. I doubt we'll be able to find a good time for everyone, but let's see what we get.

https://www.when2meet.com/?15720424-iw1rt

**Reply by michaelpj** [reply]
It has some arbitrary dates because apparently you need to do that to allow people to pick their time zones, but the intention is for this to be a generic week.

**Reply by michaelpj** [reply]
Hmm, did people fill in their timezones when filling this out? @pepeiborra aren't you in New York?

**Reply by pepeiborra** [reply]
> Hmm, did people fill in their timezones when filling this out? @pepeiborra aren't you in New York? 

I was in London, until recently. Now I am in Spain.

**Reply by July541** [reply]
> Hmm, did people fill in their timezones when filling this out? @pepeiborra aren't you in New York? 

I filled it with my timezone.

**Reply by michaelpj** [reply]
Okay good, just didn't want to pick a time that was 3am for you :D

**Reply by fendor** [reply]
cc @jneira as I believe you are interested as well if time permits it?

**Comment by drsooch** [comment]
Dang I clicked the link and regret signing in is there a way to remove me. I'd be interested in listening in.

**Comment by david-christiansen** [comment]
I would like to attend this meeting too! I filled out the when2meet.

**Comment by michaelpj** [comment]
Okay, looks like we have a winning time: 12-1pm BST. I suggest we meet for the first time next week. I'll send those of you that I have email addresses for an invitation, if you don't get one and you want one, find a way to get me your email address. I'll post the meeting link in IRC too; probably not here as it's maybe a little too easy for random scrapers to pick up :thinking:

**Reply by michaelpj** [reply]
In particular, I could not find @fendor @drsooch or @July541

**Reply by kokobd** [reply]
@michaelpj Would you mind sending me an invitation? I'd like to know what is going on recently.
